app/bundles/LeadBundle/Entity/Lead.php

```diff
@@ -668,6 +668,39 @@ public function addToPoints($points)
     }
 
     /**
+     * Subtract from current points
+     *
+     * @param $points
+     */
+    public function subtractToPoints($points)
```

Comment:
@dongilbert That would be no problem. More importantly, to you think these methods *need* to exist? I wrote them to follow the current style put the operations are so minor that I find them a bit **redundant**...